web/client/plugins/SaveAs.jsx

```diff
@@ -24,6 +24,8 @@ const showMapSaveAsSelector = state => state.controls && state.controls.mapSaveA
 /**
  * Plugin for Create/Clone a Map. Saves the map as a new Resource (using the persistence API).
  * @prop {boolean} [cfg.disablePermission=false] disable the permission selector in the tool. Can be used in context when permissions are not needed (resources are private only/using plugin with another API)
+ * @prop {object} [cfg.disablePluginIf=Boolean(expression)] disable the plugin with a custom expression that evaluates to a boolean. Can be configured in context creation or when editing a context.
+ * You may use an expression that evaluates a particular state of the program. For example: `"disablePluginIf": "{state('userrole') !== 'ADMIN'}"`. In the example, the plugin is disabled for users that do not have the role of ADMIN.
```

Comment:
This property is valid for all the plugins and already documented. We don't need to add this doc here.

```suggestion
```